app/assets/javascripts/podcasts.coffee

```diff
@@ -1,3 +1,9 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready ->
+  $('input[type=radio][name=show-radios]').change ->
```

Comment:
This can be refactored into 2 lines - add listener to all the radios, in the listener find the :checked under current group